.github/ISSUE_TEMPLATE/new-hub.yml

```
@@ -0,0 +1,164 @@
+name: 💻  New Hub
+description: Creating a new hub for 2i2c to operate
+title: "New Hub: <Hub name>"
+labels: ["type: hub"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        # Ask for a new 2i2c JupyterHub!
+        
+        This template is designed to help you provide information about your 2i2c Hub so that a 2i2c Engineer can set it up properly.
+        Answer the sections below as best you can.
+        We can update information as we learn more.
+
+        ## Community Information
+
+  - type: textarea
+    id: background
+    attributes:
+      label: Hub Description
+      description: |
+        Two or three sentences about the community and use-case for the hub.
+    validations:
+      required: true
+
+  - type: input
+    id: community-representative
+    attributes:
+      label: Community Representative
+      description: |
+        The main point-of-contact for this hub, and the person who requests changes to the hub on behalf of the community.
+        **This person will be the first Hub Administrator**, and may add new hub administrators from the JupyterHub UI.
+        Should be either a GitHub handle or email address, depending on the type of authentication you've indicated above.
```

```suggestion
        Should be either a GitHub handle or email address, depending on the type of authentication you've indicated below.
```
The auth field is actually below this one!